packages/core/src/components/forms/_controls.scss

```diff
@@ -46,10 +93,10 @@ $control-checked-background-color-active: nth(map-get($button-intents, "primary"
   }
 
   .#{$ns}-control-indicator {
-    position: absolute;
-    top: 0;
-    left: 0;
-    margin: 0;
+    display: inline-block;
+    position: relative;
+    margin-top: -3px;
```

Comment:
does this depend of font size?

Reply:
😕 it does, but checkboxes were not designed to support arbitrary font size. that would require a different approach. i've been fiddling with a flex layout a la menu items (requiring wrapping children in a `<span>`) but i don't think it's worth the change just yet.